erts/emulator/internal_doc/SuperCarrier.md

```diff
@@ -5,7 +5,7 @@ A super carrier is large memory area, allocated at VM start, which can
 be used during runtime to allocate normal carriers from.
 
 The super carrier feature was introduced in OTP R16B03. It is
-enabled with command line option +MMscs &lt;size in Mb&gt;
+enabled with command line option +MMscs <size in Mb>
```

Comment:
```suggestion
enabled with command line option `+MMscs <size in Mb>`
```

That `&lt;` does not work is a bug/feature of the markdown parser ex_doc uses. To make it work for all parsers I think it is a good idea to just make it into a code segment.

Reply:
Done